tutorials/editor_basics/editor_events.md

```diff
@@ -2,172 +2,168 @@
 title: 3. Handling Editor Events
 ---
 
-The following set of tutorials is meant to be an introduction to actions activated by editor events.
-The IntelliJ Platform SDK provides a set of callbacks for handling events related to the Editor.
+The previous tutorial [Editor Coordinate Systems](coordinates_system.md) described working with caret coordinate systems in an editor window.
```

Comment:
why refer to previous chapter?

Reply:
I add this information in case someone lands on this page, but hasn't read the previous 2 pages of the tutorial.